src/Surfnet/ServiceProviderDashboard/Application/Command/Entity/UpdateEntityIdpsCommand.php

```diff
@@ -45,6 +46,7 @@ public function __construct(
             new Assert\Type(type: IdentityProvider::class),
         ])]
         public array $institutionEntities,
+        public Contact $applicant,
```

Comment:
readonly?

Reply:
Feels weird with a public field, but it should only be read. So yeah! good suggestion